### DISCUSSION TITLE
Escaping in Configure Filters

### DISCUSSION THREAD

**Discussion by namtab00** [dbeaver/dbeaver#8767] [Q&A] [answered]
#### System information: 
- Win10
- DBeaver Version 7.0.5.202005171753

#### Connection specification:
- irrelevant

#### Describe your question:
How do I escape an underscore "_" in a filter inclusion / exclusion, as per [Configure Filters](https://github.com/dbeaver/dbeaver/wiki/Configure-Filters) ?

I'm trying to exclude from the tree all functions starting with _, so I tried:

- _*
- \\_*
- [_]*

but none seems to work...

**Comment by uslss** [comment, marked as answer]
To escape underscore you could use backslash, e.g. %\_%

**Reply by tSpiny** [reply]
It doesn't work at all. I have 2 schemas in my db:  
- `dwh2xyz`  
- `dwh_xyz`

I tried to filter only `dwh_xyz` with `dwh\_*` . And it didn't work.

**Reply by ivenspontes** [reply]
save my day, thanks

**Reply by mateusbandeiraa** [reply]
Note for future readers: @uslss's example uses percent symbols instead of slashes. `%_%` didn't work for me, but `\_` did.
